public/src/hocs/withS3Data.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
I was thinking that we could reuse this. I think it would be quite straight forward to get the banner deploy dashboard using this. Might even be possible to get the banner/epic editors using this too.

Reply:
Interesting, it feels like a more general form of `TestEditor`?

Reply:
Yeah exactly!